src/en/mangadistrict/src/eu/kanade/tachiyomi/extension/en/mangadistrict/MangaDistrict.kt

```diff
@@ -90,6 +90,13 @@ class MangaDistrict :
         }.let(screen::addPreference)
     }
 
+    override fun imageFromElement(element: Element): String? {
+        return when {
+            element.hasAttr("data-wpfc-original-src") -> element.attr("data-wpfc-original-src")
```

Comment:
use `absUrl`

Reply:
Changed it so it's prefixed with `abs:`, consistent with the super method.

Out of curiosity, when would you use `absUrl` over prefixing the attribute with `abs:`? Personal preference?

Reply:
both are essentially the same, but I think `absUrl` is more clear imo in terms that it is accessing absolute url from the given attribute, whereas `abs:attr` can be misinterpreted for being a literal attribute.